### PR TITLE
Update Configure a resource account, plan CAA and plan CQ articles

### PR DESCRIPTION
**Pull request by dariomws**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4384
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/4385

@CarolynRowe or @jambirk could you kindly help with these articles? I think step 1 "Port or get a toll or toll-free service number" should be optional if you will assign a Direct Routing or hybrid number.
Also, in the step 6 when you run Set-CsHybridApplicationEndpoint and Set-CsOnlineApplicationInstance you should use the same number.

### CODE REVIEW

**Comment by dstrome** [comment]
@CarolynRowe - can you confirm these changes are correct? I'm unsure of the nuances here.

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This inherently means that you will have a hybrid implementation of Skype for Bu
 An Auto attendant is a cloud service that accept customer calls and play greetings, provide them with menu options, and interact with callers using speech or the dialpad to route their calls to the right destination. Each auto attendant is assigned a **resource account** (see[Configure resource accounts](configure-onprem-ra.md)) on your Skype for Business Server 2019 system that will be linked directly to an auto attendant in the Microsoft Teams admin center. See [What are Cloud auto attendants?](/SkypeForBusiness/what-is-phone-system-in-office-365/what-are-phone-system-auto-attendants.md) for more detail on what auto attendants are and what options and features exist for auto attendants.
```
Comment (JohanFreelancer9) [inline Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md:23]:
```suggestion
An auto attendant is a cloud service that accepts customer calls and plays greetings, provides them with menu options, and interacts with callers using speech or the dial pad to route their calls to the right destination. Each auto attendant is assigned a **resource account** (see [Configure resource accounts](configure-onprem-ra.md)) on your Skype for Business Server 2019 system that will be linked directly to an auto attendant in the Microsoft Teams admin center. See [What are Cloud auto attendants?](/SkypeForBusiness/what-is-phone-system-in-office-365/what-are-phone-system-auto-attendants.md) for more detail on what auto attendants are and what options and features exist for auto attendants.
```

Comment (JohanFreelancer9) [reply Skype/SfbHybrid/hybrid/plan-cloud-auto-attendant.md:23]: Also, note that the link for "What are Cloud auto attendants" does not seem to work. Please check.

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**